MANIFEST.in

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Yes, that was a _feature_.

Who builds distribution tarballs anyway? Developers do. And developers have checkouts. :stuck_out_tongue: 

Consider that if git was more prominently around when the whole Python packaging debacle became known as setuptools, we probably [wouldn't be using MANIFEST.in in the first place](https://pythonhosted.org/setuptools/setuptools.html#generating-source-distributions)!